src/main/java/seedu/address/model/candidate/ApplicationStatus.java

```diff
@@ -5,10 +5,10 @@
     public static final String MESSAGE_CONSTRAINTS =
             "Application Status should be Pending, Accepted or Rejected";
     public static final String PENDING_STATUS = "Pending";
-    public static final String ACCPETED_STATUS = "Accepted";
+    public static final String ACCEPTED_STATUS = "Accepted";
```

Comment:
thanks for catching my english mistake !! appreciate it!

Reply:
just a minor typo 😛